content/projects/open-alias/index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "OpenAlias"
+date: 2018-07-09T11:41:44+12:00
+draft: false
+author: "Alex Vazquez"
+description: "NAV OpenAlias allows you to register a free OpenAlias address and link it to your real NavCoin address using a unique 'email style' user name."
+resources: []
+filter: ["Open Alias"]
+bg_image: "/images/icons/icon-purple-lightbulb-new.png"
+slug: ""
+type: "project"
+feature_image: ""
+show_title_text: false
+in_progress: false
+completed: true
+show_on_roadmap: true
+show_on_projects: true
+allow_click_through: true
+percent_complete: 100
+github_url: "https://github.com/aguycalled/openalias-register"
+project_url: "http://openalias.nav.community"
+reddit_url: ""
+twitter_url: ""
+card_color: "#7d5ab5"
+font_color: "#fff"
+weight: 100
+---
+
+{{< project_header
+    title="NavCoin"
+    subhead="OpenAlias Registration"
+    text="OpenAlias has been implemented to make payments to NavCoin wallets easier. It’s another example of the ways that NavCoin is simplifying cryptocurrency."
+    captionPadding="80px"
+    imgSrc="/images/icons/nav_blue-peertopeer-large.png"
+    imgHeight="300px"
+    imgPadding="20px"
+    bgColor="#FFF"
+    bgImgSrc=""
+    fontColor="#333">}}
+
+{{< center_text_section
+    title="What is OpenAlias?"
+    bgColor="#42b6eb"
+    gradient=""
+    fontColor="#FFF">}}
+    <div style="width: 80%; margin: 0 auto;">
+    <p>OpenAlias (openalias.org) is an open standard that allows users to transform a lengthy alphanumeric wallet address into a form that’s much easier to read, write, and remember. For example, yourname@nav.community</p>
+    </div>
+{{< /center_text_section >}}
+
+<br />
+<section class="container">
+
+## Overview
+
+NAV OpenAlias is a little like PayPal, which lets you send a payment to an email address. The difference is that OpenAlias uses an easy to remember 'email style' address to represent your full NavCoin wallet address.
+
+For more information, visit [openalias.org](https://openalias.org/)
+
+## Registering for an NavCoin Open Alias
+
+First, visit [http://openalias.nav.community](http://openalias.nav.community).
+
+### 1. Register your name
+
+Register a name you like and the suffix `@nav.community` will be added automatically.
+
```

Comment:
Seems to me we should add info explaining how user can generate new wallet address to be used for the new open alias name they're creating. Info about how you can keep your staking address separate from your open alias address(es). Just a thought... Or we just leave it to the 'instructions' they'll get on the actual page. Am fine w/ either.

Reply:
I think we remove the instructions and they follow that on the actual page at time of creation.

Reply:
sounds fine to me